doc/spec.md

```diff
@@ -2554,14 +2554,18 @@ Object literals are extended to support type annotations in methods and get and
 &emsp;&emsp;&emsp;*PropertyName*&emsp;*CallSignature*&emsp;`{`&emsp;*FunctionBody*&emsp;`}`  
 &emsp;&emsp;&emsp;*GetAccessor*  
 &emsp;&emsp;&emsp;*SetAccessor*
+&emsp;&emsp;&emsp;`...`*SpreadAssignment*
 
 &emsp;&emsp;*GetAccessor:*  
 &emsp;&emsp;&emsp;`get`&emsp;*PropertyName*&emsp;`(`&emsp;`)`&emsp;*TypeAnnotation<sub>opt</sub>*&emsp;`{`&emsp;*FunctionBody*&emsp;`}`
 
 &emsp;&emsp;*SetAccessor:*  
 &emsp;&emsp;&emsp;`set`&emsp;*PropertyName*&emsp;`(`&emsp;*BindingIdentifierOrPattern*&emsp;*TypeAnnotation<sub>opt</sub>*&emsp;`)`&emsp;`{`&emsp;*FunctionBody*&emsp;`}`
 
-The type of an object literal is an object type with the set of properties specified by the property assignments in the object literal. A get and set accessor may specify the same property name, but otherwise it is an error to specify multiple property assignments for the same property.
+The type of an object literal is an object type with the set of properties specified by the property assignments in the object literal, plus any properties from spread assignments.
+A get and set accessor may specify the same property name, but otherwise it is an error to specify multiple property assignments for the same property unless they come from spread assignments.
```

Comment:
This is a little unnecessary because spread assignments don't actually introduce property assignments. Maybe something like the following would be better

```
A get and set accessor may specify the same property name, and spread assignments may contain properties that have already been declared by other members in the object literal, but otherwise it is an error to specify multiple property assignments for the same property.
```